zerver/openapi/zulip.yaml

```diff
@@ -14442,6 +14457,18 @@ paths:
                           client capability and access the `user_settings` object instead.
 
                           [capabilities]: /api/register-queue#parameter-client_capabilities
+                      receives_typing_notifications:
+                        type: boolean
+                        description: |
+                          Whether the user is configured to receive typing notifications from other
+                          users. The server will only deliver typing notifications events to users who
+                          for whom this is enabled.
+
+                          By default, this is set to true, enabling user to receive typing
+                          notifications from other users.
+
+                          **Changes**: New in Zulip 9.0 (feature level 253). Previously, there were
+                          only options to disable sending typing notifications.
```

Comment:
These changes entries should appear in all the places in this file that the field is mentioned -- it looks like it's missing in a couple places.